pkg/shell/base_index.js

```diff
@@ -613,6 +625,51 @@ function Index() {
         return false;
     };
 
+    function setDarkMode(documentElement, dark_mode) {
+        if (dark_mode) {
+            documentElement.classList.add('pf-theme-dark');
+        } else {
+            documentElement.classList.remove('pf-theme-dark');
+        }
+    }
```

Comment:
These 7 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17914-20221130-135116-003cefd9-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)